npm-publish/README.md

```diff
@@ -9,6 +9,7 @@ The `npm-prepare-release` action is meant to be called first and will create a p
 * `NPM_TOKEN`: (required) the npm token used to publish the package.
 * `GH_TOKEN`: (required) the GitHub access token to use. It is recommended to use the standard GitHub Actions access token (used in example).
 * `node-version`: (optional) the Node.js version to use for the Action.
+* `provenance`: (optional) set to `true` to generate provenance statement for the published package. Requires the `id-token: write` permission.
```

Comment:
Maybe we should use `PROVENANCE`? Makes it slightly simpler to have both in the same case.

```suggestion
* `PROVENANCE`: (optional) set to `true` to generate provenance statement for the published package. Requires the `id-token: write` permission.
```